PEP 1: Further clarify PEP-Delegate notification & acceptance process (#2273)
python/peps: pep-0001.txt

The final authority for PEP approval is the Steering Council. However, whenever
a new PEP is put forward, any core developer who believes they are suitably
experienced to make the final decision on that PEP may offer to serve as its
PEP-Delegate by `notifying the Steering Council <Steering Council issue_>`_

> of their intent. If the Steering Council accepts their offer,

Reviewer: See suggestion below for narrative on "approves" over "accepts"

```suggestion
of their intent. If the Steering Council approves their offer,
```

Any legacy references to "BDFL-Delegate" should be treated as equivalent to
"PEP-Delegate".

> Individuals offering to serve as PEP-Delegate should notify the PEP authors
> and PEP sponsor of their intent to self-nominate, and submit their request as

Reviewer: I'm getting to a point of cognative dissonance with "PEP" in this sentence! Perhaps:
```suggestion
An individual offering to serve as PEP-Delegate must notify the relevant 
authors and sponsor (where a sponsor exists) for the PEP. The request must be 
formally submitted by opening
```

Reviewer: This could be combined with @brettcannon's suggestion that I just saw.

Author: Yeah; I'll have to do so manually

Author: Done

> `a new issue <Steering Council issue_>`_ on the `Steering Council repository`_.

Author: Sidenote, this eliminates the need for the SC repo link target, so I'll need to delete that manually.

Those taking on this responsibility are free to seek
additional guidance from the Steering Council at any time, and are also expected
to take the advice and perspectives of other core developers into account.

> The Steering Council will generally accept such self-nominations by default,
> but may choose to decline them.

Reviewer: ```suggestion
The Steering Council will ordinarily approve such self-nominations,
but may choose to decline them.
```

If "ordinarily" is too formal, "usually" also works.

"approve" makes more explicit that you must hear back in the affirmative from the Council.

A

Author: I see what you're trying to do here, but I'm concerned it is getting a bit far away from the original intended meaning—it now implies that the SC will typically approve nominations, but no longer directly mentions the decision-making rationale of approve-by-default, the original purpose of this sentence. It is now making a statement on approval _probabilty_ (offers will be ordinarily approved) rather than approval _strategy_ (offers will be approved by default unless there is a significant reason not to do so), as "ordinarily" (and "usually") refers to frequency while "generally" more generically is a general hedge, while "accept [...] by default" was what "generally" was referencing, not "accept" alone.

A more conservative suggestion that keeps the original meaning but applies your "approve" change would be the following:

```suggestion
The Steering Council will generally approve such self-nominations by default,
but may choose to decline them.
```

Author: Also, @brettcannon, thoughts?

Reviewer: Thanks for the added rationale -- the "generally" and "by default" felt odd in the same sentence, but I see they are serving different purposes.

Any legacy references to "BDFL-Delegate" should be treated as equivalent to
"PEP-Delegate".

An individual offering to nominate themselves as a PEP-Delegate must notify
the relevant authors and (when present) the sponsor for the PEP, and submit
their request to the Steering Council

> (which can be done via a `new issue <Steering Council issue_>`_ ).

Reviewer: ```suggestion
(which can be done via a `new issue <Steering Council issue>`_ ).
```

I don't believe the first underscore in the link target is required.

I'd also probably say ``request to the Steering Council through a `new issue <...>`_`` but at this point marginal benefit is quickly decreasing, so I won't formally suggest it!

Author: > I don't believe the first underscore in the link target is required.

It is here, since otherwise it is read as a literal URL rather than a target name and the link breaks (unlike `:ref:`, `:doc:`, etc directives that use an otherwise very similar syntax where it isn't required, since the content of `<>` must be an implicit or explicit target name, not a literal URL).

> I'd also probably say request to the Steering Council through a `new issue <...>`_ but at this point marginal benefit is quickly decreasing, so I won't formally suggest it!

I also preferred that wording and originally implemented something very close to it, but ultimately deferred to adhering pretty closely to what @brettcannon suggested above.

.. _`GitHub issue`: https://github.com/python/peps/issues

.. _Steering Council repository: https://github.com/python/steering-council

> .. _Steering Council issue: https://github.com/python/steering-council/issues/new/choose

Reviewer: That repo doesn't have an issue chooser set up, I think (the link redirects to `https://github.com/python/steering-council/issues/new`)

```suggestion
.. _Steering Council issue: https://github.com/python/steering-council/issues/new
```

Author: Yes, but this is the canonical link for the <kbd>New Issue</kbd> button in GitHub's UI, and will automatically redirect to the chooser if one is set up (whereas cutting out `/choose` would ignore it even if it was set up, which is not likely what we want here), so the link should be kept how it is for forward compat and consistency with GitHub's UI, instead of trying to second-guess it.
